.github/workflows/build.yml

```diff
@@ -121,8 +123,21 @@ jobs:
             ${{ matrix.stack }} deploy.sh --overwrite \
             extensions/no-debug-non-zts-${{ matrix.series }}/relay-0.4.4
 
-      - name: Build swoole extension
-        if: matrix.stack != 'heroku-22' || matrix.series >= 20210902
+      - name: Build swoole 4.x extension
+        # skip build on heroku-22
+        if: matrix.stack != 'heroku-22'
```

Comment:
So there will be no swoole 4.x neither swoole 5.x for someone on heroku-22 with PHP 8.0 ?

Reply:
Have you tried using 8.0 on `heroku-22`?

Reply:
yep sorry I realized later your other comment - heroku-22 is only php >= 8.1